Makefile

```diff
@@ -18,7 +17,7 @@ build:
 env:
 	$(PIPENV_CMD) --venv || \
 	(echo "Creating virtual environment .venv"; \
-	PIPENV_VENV_IN_PROJECT=1 $(PIPENV_CMD) install --python $(PYTHON);)
+	PIPENV_VENV_IN_PROJECT=1 $(PIPENV_CMD) sync;)
```

Comment:
wondering, how is the python version chosen after this change? does it relies on the already installed version?

Reply:
It relies on the version set at the `pipfile`:
```
[requires]
python_version = "3.6"
```